bundle/get_started.md

```diff
@@ -314,15 +318,17 @@ There are several predefined scripts in MONAI bundle module, here we leverage th
 Just define the entry point expressions in the config to execute in order, and specify the `runner_id` in CLI script.
```

Comment:
This line description seems outdated.
Maybe you can describe what are the "initialize", "run" and "finalize" sections here.

Thanks.